pkgs/misc/cbeams/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, blessings, docopt }:
+
+buildPythonPackage rec {
+  pname = "cbeams";
+  version = "1.0.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0y5q2d4kfrlihj00a4sfwjn9g6a0bwlmnv22xssyldyhlkj1p0aj";
+  };
+
+  propagatedBuildInputs = [ blessings docopt ];
+
+  postPatch = ''
+    sed -i "s/docopt==0.6.1/docopt>=0.6.1/; s/blessings==1.6/blessings>=1.6/" setup.py
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/tartley/cbeams";
+    description = "Command-line program to draw animated colored circles in the terminal";
```

Comment:
is this just meant to be used as an application (not a python library)?

Reply:
Yes, it's _just_ an application. I have changed the misleading `buildPythonPackage` function to `buildPythonApplication`.